model/service.go

```diff
@@ -37,12 +42,28 @@ type ServiceDiscovery interface {
 type Service struct {
 	// Name of the service
 	Name string `json:"name"`
+
 	// Namespace of the service name, optional
 	Namespace string `json:"namespace,omitempty"`
+
 	// Tags is a set of declared distinct tags for the service
 	Tags []Tag `json:"tags,omitempty"`
+
 	// Ports is a set of declared network service ports
-	Ports []Port `json:"ports,omitempty"`
+	Ports []*Port `json:"ports,omitempty"`
+
+	// Addresses lists service IPv4 addresses
+	Addresses map[string]bool `json:"addresses"`
```

Comment:
I added it in preparation for TCP services for which host field, the basis for HTTP routing, is not available. This adds support for "curl cluster_ip" instead of "curl cluster_name" for HTTP services.